apps/dotcom/src/components/ShareMenu.tsx

```diff
@@ -43,7 +44,7 @@ function isSharedReadonlyUrl(pathname: string) {
 }
 
 function isSharedReadWriteUrl(pathname: string) {
-	return pathname.startsWith('/r/')
+	return pathname.startsWith(`${ROOM_PREFIX}/`)
```

Comment:
Kinda undecided whether I like that we include the leading slash. This showcases why. We need to add the trailing slash, but not the leading one. It does save some typing, but makes me always think if I need to include it or not, so it might be slightly confusing. WDYT?

Reply:
I'm down to get rid of the slash - yeah, I see your point on this 👍 
Personally was torn as well so it's a fair point!